packages/apps/desktop-viewer-test/src/backend/UserSettings.ts

```diff
@@ -93,6 +93,19 @@ class UserSettings {
     }
     this._writeSettings();
   }
+
+  public removeRecent(file: ViewerFile) {
+    if (this.settings.recents) {
+      this.settings.recents = this.settings.recents.filter(
+        (existingFile) =>
+          (file.path && existingFile.path !== file.path) ||
+          (file.iModelId &&
```

Comment:
This filter callback function is the same as the findIndex callback function above and can be extracted.